src/utils/some-base.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@stencil/core';
+
+@Component({
```

Comment:
is it important that the class has a decorator? I can not inherit components from simple classes?

Reply:
Unfortunately, that is the case for now.